cargo-shuttle/src/factory.rs

```diff
@@ -0,0 +1,11 @@
+use async_trait::async_trait;
+use shuttle_service::Factory;
+
+pub struct LocalFactory {}
+
+#[async_trait]
+impl Factory for LocalFactory {
+    async fn get_sql_connection_string(&mut self) -> Result<String, shuttle_service::Error> {
+        todo!()
```

Comment:
If we want to break this down into two PRs for velocity, we could just `Err` out here and say services with database dependencies are not (yet) supported by `cargo shuttle run` - try deploying instead!